plugins/proxmox/proxmox_vm_count

```diff
@@ -4,34 +4,35 @@
 : << EOF
 =head1 NAME
 
-proxmox_vm_count - Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
+proxmox_vm_count
+
+=head1 NOTES
+
+Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
 
 =head1 CONFIGURATION
 
-This plugin needs to run as the root user in order to have permission to run lvs and vgs
+This Plugin has to be run as root:
+
   [proxmox_vm_count]
-  [proxmox_count]
-  user root
-  group root
+    user root
+    group root
 
-=head1 AUTHOR
+=head1 MAGIC MARKERS
 
-Michael Grote
+  #%# family=auto
+  #%# capabilities=autoconf
 
+=head1 AUTHOR
+
+Michael Grote - git.mgrote.net
 
 =head1 LICENSE
 
 GPLv3 or later
 
 SPDX-License-Identifier: GPL-3.0-or-later
 
-=head1 MAGIC MARKERS
```

Comment:
Currently it is quite well established, that the `MAGIC MARKERS` section (being the least interesting one for human readers) is the *last* one.

I would prefer to keep it like that, if you do not have specific reasons for changing this.

Reply:
fixed